README.md

```diff
@@ -31,6 +31,17 @@ The *ArcGIS Runtime Toolkit for iOS* has a *Target SDK* version of *12.0*, meani
 
 ## Instructions
 
+### Swift Package Manager
+
```

Comment:
I tested this in a forked repo and it works ok, I don't think this can be tested here until the SPM PR is merged because the package manifest must be in the root of the repo on the main branch.